src/autocomplete.js

```diff
@@ -257,6 +285,30 @@ class Autocomplete extends Component {
 
   //////////////////////////// BUTTON
 
+  buttonKeyDownHandlers = {
+    ArrowDown: this.keyDownHandlers.ArrowDown,
+
+    ArrowUp: this.keyDownHandlers.ArrowUp,
+
+    Enter(event) {
```

Comment:
Totally 👍 I did that wrong. I meant for space to handle that functionality :) that's seems to be how the default browser select works.